File: app/src/main/java/com/worldventures/dreamtrips/wallet/service/nxt/model/MultiRequestElement.java
package com.worldventures.dreamtrips.wallet.service.nxt.model;

import com.google.gson.annotations.SerializedName;

import org.immutables.gson.Gson;
import org.immutables.value.Value;

@Gson.TypeAdapters
@Value.Immutable
public interface MultiRequestElement {

   @SerializedName("Operation")
   String operation();
   @SerializedName("TokenName")
   String tokenName();
   @SerializedName("Value")
   String value();
   @SerializedName("RefId")
   String referenceId();

}

File: wallet/src/main/java/com/worldventures/wallet/service/nxt/model/MultiRequestElement.kt
package com.worldventures.wallet.service.nxt.model

import com.google.gson.annotations.SerializedName

data class MultiRequestElement(
      @field:SerializedName("Operation") val operation: String,
      @field:SerializedName("TokenName") val tokenName: String,
      @field:SerializedName("Value") val value: String,
      @field:SerializedName("RefId") val referenceId: String
)